open-craft/opencraft
# docs: Add marketing app usage instructions

Adds documentation for the marketing app.

### Related Tickets
- [BB-4202](https://tasks.opencraft.com/browse/BB-4202)

### Testing Instructions
1. Checkout to this branch
2. Run `make docs-serve` to start the local documentation server.
3. Visit http://localhost:5001/marketing_usage.html
4. Verify that the documentation contains the information from [this document](https://docs.google.com/document/d/1-gOBH7BbiwfpHlpxRzSwJsYOa5xe1L3NmNd49X7XUnA/edit#)

### Author Notes
1. I have removed two images from document as they were just admin panel images and were redundant.
